vercel/resource_alias.go

```diff
@@ -119,7 +119,7 @@ func (r resourceAlias) Read(ctx context.Context, req tfsdk.ReadResourceRequest,
 
 	out, err := r.p.client.GetAlias(ctx, state.UID.Value, state.TeamID.Value)
 	var apiErr client.APIError
-	if err != nil && errors.As(err, &apiErr) && apiErr.StatusCode == 404 {
+	if err != nil && (errors.As(err, &apiErr) && apiErr.StatusCode == 404) || (err != nil && err.Error() == "no content") {
```

Comment:
I'd put a TODO here too. Then we can clean it up when they fix their API.

Not critical, just makes clean up easier later.